src/components/elements/side-tab.tsx

```diff
@@ -168,37 +168,38 @@ const SideTab = ({
     const [active_tab, setActiveTab] = useTabStateQuery(getTabs(children))
     const [is_menu, setMenu] = useState(false)
 
-    const Tabs = (props) => {
-        return React.Children.map(children, (child) => {
-            const { label, text, onClick } = child.props
-
-            if (!React.isValidElement(child)) {
-                return <></>
-            }
-
-            return (
-                <div>
-                    <Tab
-                        font_size={font_size}
-                        mobile={props.is_mobile}
-                        text={text}
-                        line_height={line_height}
-                        opacity={opacity}
-                        onClick={(e) => {
-                            if (onClick) {
-                                onClick(e)
-                            }
-
-                            setActiveTab(e)
-                            setMenu(!is_menu)
-                        }}
-                        active_tab={active_tab}
-                        label={label}
-                    />
-                </div>
-            )
-        })
-    }
+    const Tabs = () => (
```

Comment:
@niloo-fs You should wrap `Tabs` with `memo`, Not `SideTab`.

Reply:
My bad, will change this